src/include/sof/lib/dai.h

```diff
@@ -85,6 +85,7 @@ struct dai_ops {
 	int (*probe)(struct dai *dai);
 	int (*remove)(struct dai *dai);
 	uint32_t (*get_init_delay_ms)(struct dai *dai);
+	uint32_t (*get_fifo_depth)(struct dai *dai, int direction);
```

Comment:
why not just `int`?

Reply:
yeah was just trying to be smart, maybe just change to int. I guess the depth can't be negative. so problem with returning a value and error code with same return...

Reply:
so int it is